content/api_chef_server.md

```diff
@@ -931,26 +938,33 @@ where:
 -   `username` must begin with a lower-case letter or digit, may only
     contain lower-case letters, digits, hyphens, and underscores. For
     example: `chef`.
+    `username` is required to be present and have a valid value. A valid
```

Comment:
Should "name" on line 924 be changed to "username" or should this be "name"?

Reply:
yes 924 should be username.  Thank you for catching that.  Where I have chefserver in the URIs it would be more consistent to use chef.example.  I'll fix and repost.

Reply:
reposted

Reply:
Could you add your DCO signoff too. Thanks!

Reply:
Done.